src/app/screens/TimesheetScreen/view/employeeList.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useCallback, useState} from 'react';
+import {FlatList, StyleSheet, TouchableOpacity, View} from 'react-native';
+
+import {useNavigation} from '@react-navigation/native';
+import {MainScreenNavigationProp} from '../../../navigation/types';
+
+import EmployeeCard from '../component/employeeCard';
+import Input from '../../../components/input';
+import DateRange from '../../../components/pickers/dateRange';
+import Linear from '../../../components/seperator/linear';
+import EmptyList from '../component/emptyList';
+
+import {Employee} from '../interface';
+
+import {Calendar, Search} from '../../../constant/icons';
+import sizes from '../../../constant/sizes';
+import {employeeData} from '../../../constant/timesheet';
+import {USER_TIMESHEET} from '../../../constant/screenNames';
+
+type Props = {
+  item: Employee;
+};
+
+const seperator = () => <Linear />;
+const footer = () => <Linear style={styles.footer} />;
+const emptyComponent = () => <EmptyList />;
+const searchIcon = () => <Search style={styles.icon} />;
+
+const EmployeeList = () => {
+  const [isVisible, setIsVisible] = useState<boolean>(false);
+  const [filterData, setFilterData] = useState<Employee[]>(employeeData);
+  const navigation = useNavigation<MainScreenNavigationProp>();
+
+  const toggelDatePicker = () => setIsVisible(v => !v);
+
+  const onDateRangeSubmit = useCallback((startDate?: Date, endDate?: Date) => {
+    console.log(startDate, endDate);
+  }, []);
```

Comment:
what's after onDateRangeSubmit?

Reply:
api call to featch employee list by date range

Reply:
so add a TODO comment here